go/vt/vtadmin/api.go

```diff
@@ -313,6 +313,7 @@ func (api *API) Handler() http.Handler {
 	router.HandleFunc("/clusters", httpAPI.Adapt(vtadminhttp.GetClusters)).Name("API.GetClusters")
 	router.HandleFunc("/gates", httpAPI.Adapt(vtadminhttp.GetGates)).Name("API.GetGates")
 	router.HandleFunc("/keyspace/{cluster_id}", httpAPI.Adapt(vtadminhttp.CreateKeyspace)).Name("API.CreateKeyspace").Methods("POST")
+	router.HandleFunc("/keyspace/{cluster_id}/topology", httpAPI.Adapt(vtadminhttp.GetTopology)).Name("API.GetTopology")
```

Comment:
why is this within /keyspace? keyspaces exist _within_ topologies, not the other way around

Reply:
That was the prior art for "cluster specific" `cluster_id` things - but I totally agree! I'd rather have `/clusters/cluster_id/topology` for vtadmin.